### PR TITLE
feat: integrate zendesk support

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
Closes https://app.clickup.com/t/31582pd

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/styles/zendesk.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#launcher {
```
Comment (ItsANameToo) [inline src/styles/zendesk.css:1]: use tailwind classes, especially for the colours

File: src/app/contexts/Zendesk/Zendesk.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { useAccentColor, useTheme } from "@/app/hooks";
+import React from "react";
+import Zendesk, { ZendeskAPI } from "react-zendesk";
+const ZENDESK_KEY = "0e4c4d37-9d38-4be4-925d-e659dd4d12bd";
```
Comment (ItsANameToo) [inline src/app/contexts/Zendesk/Zendesk.tsx:4]: make it a `.env` variable so it can be updated without a release

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Minor bug. Zendesk form can display incorrect styling when the wallet theme is changed.

- Sign into wallet with a profile that has light mode enabled by default
- Change theme to dark mode
- Open zendesk form
- See it's in light mode

(or vice versa)

image
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
The 'Go Back' button doesn't appear to be styled correctly in dark mode
image

File: src/app/constants/navigation.ts [hunk 3]
```diff
@@ -43,11 +43,17 @@ export const getUserMenuActions = (t: TFunction): (DropdownOption & NavigationBa
...
-		label: t("COMMON.SUPPORT"),
+		label: t("COMMON.DOCUMENTATION"),
 		mountPath: () => "https://arkvault.io/docs",
 		title: "support",
 		value: "support",
 	},
+	{
+		label: t("COMMON.CONTACT_US"),
+		mountPath: () => `/`,
+		title: "sign-out",
```
Comment (ItsANameToo) [inline src/app/constants/navigation.ts:54]: sus title

File: src/styles/zendesk.css (hunk 1 above)
Comment (goga-m) [reply src/styles/zendesk.css:1]: @ItsANameToo removed hardcoded values and applied tw classes :+1:

**Comment by goga-m** [comment]
@samharperpittam both UI/UX issues should be fixed. When we save in settings, the chat widget should hide and show with the new colors.

**Review by samharperpittam — APPROVED** [review]

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**